### ISSUE TITLE
Surnames not in alphabetical order (MikeNL)

### ISSUE THREAD

**Issue by FreeUKGenIssues** [FreeUKGen/FreeUKRegProductIssues#531]
Issue reported by **MikeNL** at 2015-10-23 12:34:38 UTC
Time: 2015-10-23T12:21:48+00:00
Session ID: 5004fa876e8cffef284fbf58d0c39785
Problem Page URL: [/userid_details/selection?option=Select%20specific%20surname/forename](/userid_details/selection?option=Select%20specific%20surname/forename)
Previous Page URL: [http://freereg2.freereg.org.uk/userid_details/options](http://freereg2.freereg.org.uk/userid_details/options)
Reported Issue:
It is not obvious that the userids are in the order A to Z and then a to z. I initially missed the second set.

Would it not be better to sort userids irrespective of case?    

![Screenshot](http://www.freereg.org.uk)

**Comment by edickens** [comment]
I have changed all the surnames that started in lowercase to start in uppercase.  There was one "van" but I have put that in uppercase.
However there are five Profile Edits that do not work.  Select "List Profiles by Lastname" and scroll to the bottom to find them.  Need fixing by Kirk and if it is the Syndicate that is the problem (I have found a few with no syndicate and fixed those) then use "Unknown".  E
